rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.create/20_synthetic_source.yml

```diff
@@ -990,3 +970,247 @@ subobjects auto:
   - match: { hits.hits.3._source.id: 4  }
   - match: { hits.hits.3._source.auto_obj.foo: 40 }
   - match: { hits.hits.3._source.auto_obj.foo\.bar: 400 }
+
+---
+synthetic_source with copy_to:
+  - requires:
+      cluster_features: ["mapper.source.synthetic_source_restrictions_removed"]
+      reason: requires copy_to support in synthetic source
+
+  - do:
+      indices.create:
+        index: test
+        body:
+          mappings:
+            _source:
+              mode: synthetic
+            properties:
+              number:
```

Comment:
Should we test `copy_to` in combination with `ignore_above` and `ignore_malformed`?

Reply:
We test it with `ignore_malformed` for `unsigned_long` f.e.

Reply:
`ignore_above` is a good point, i'll follow up.